packages/launchpad-auth/README.md

```diff
@@ -0,0 +1,19 @@
+# @cumulus/launchpad-auth
+
+Utilities for authentication with Launchpad.
```

Comment:
Nit: 

```suggestion
Utilities for authentication by Cumulus using Launchpad.
```

Or something to specify that this isn't a general-purpose auth lib for Launchpad.    Unless it is.